internal/cmd/server.go

```diff
@@ -32,7 +32,7 @@ var (
 func getServerCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "server",
-		Aliases: []string{"serve"},
+		Aliases: []string{"c"},
```

Comment:
what's happening here

Reply:
should not be changed, thanks for catching fixed